core/commands/repo.go

```diff
@@ -152,3 +156,56 @@ RepoPath        string the path to the repo being currently used
 		},
 	},
 }
+
+var repoFsckCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Removes repo lockfiles",
+		ShortDescription: `
+'ipfs repo fsck' is a plumbing command that will remove repo and level db lockfiles.
+`,
+	},
+	Run: func(req cmds.Request, res cmds.Response) {
+		n, err := req.InvocContext().GetNode()
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+		if !n.OnlineMode() {
+			res.SetError(cmds.ErrNodeOffline, cmds.ErrNormal)
+			return
+		}
+
+		configRoot := req.InvocContext().ConfigRoot
+
+		dsPath, err := config.DataStorePath(configRoot)
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
+		dsLockFile := filepath.Join(dsPath, "LOCK") // TODO: get this lockfile programmatically
+		repoLockFile := filepath.Join(configRoot, lockfile.LockFile)
+
+		log.Infof("Removing repo lockfile: %s", repoLockFile)
+		log.Infof("Removing datastore lockfile: %s", dsLockFile)
+
+		err = os.Remove(repoLockFile)
+		if err != nil {
```

Comment:
so, one here for both of these removes that we are okay with getting is the 'file does not exist' type errors. The `os` package gives us a nice way to check this:
```go
if err != nil && !os.IsNotExist(err) {
  res.SetError(err, cmds.ErrNormal)
  return
}
```

If the lockfiles don't exist, we're just fine with that, its not an error.